src/core/core.controller.js

```diff
@@ -202,7 +202,7 @@ function isDomSupported() {
 function getCanvas(item) {
 	if (isDomSupported() && typeof item === 'string') {
 		item = document.getElementById(item);
-	} else if (item.length) {
+	} else if (Array.isArray(item) && item.length) {
```

Comment:
This is a bug! 
We have a helper for isArray (with fallback implementation, not sure for what platform)
So I'd import `isArray` from `'helpers.core'` and use that or just do plain `item && item.length`.

Reply:
I'll probably import isArray from helpers.core because item && item.length would trigger an error if item.is still not an array.

Reply:
Actually you should not, because NodeList is not strictly an array, but it does work here.
I wonder why we don't have a test for that, eg: 
```js
var canvas = document.getElementsByClassName('canvas');
var chart = new Chart(canvas, {...});
```

[ref](https://stackoverflow.com/questions/21714808/array-isarray-does-not-work-on-a-nodelist-is-there-an-alternative)

Reply:
Ok so the type i should check is html collection then?

Reply:
It can be array, HTMLCollection, NodeList, or maybe something else for some other framework. So to keep it working as it was, but avoid the 'length of null' error, `item && item.length` works fine.

Reply:
Actually, there is a test of an array in the ctx->
![image](https://user-images.githubusercontent.com/34402401/93087530-10178100-f66f-11ea-9cdb-c492d70bdee1.png)

So i don't think it was intended for that use, i think that isArray is sufficient to fix the bug in this pr. Maybe if we want to support an html collection, we should open another pr in wich we modify the tests. Maybe adding an issue would be okay.

Reply:
It's true, it will only throw an exception when .lenght is called in null or undefined. I'll fix it!

Reply:
Now the error will be triggered if the item doesn't understand lenght. Which is already more expresive.

Reply:
It won't say null or undefined that's why